c7n/reports/csvout.py

```diff
@@ -278,7 +278,7 @@ def get_records(bucket, key, session_factory):
     custodian_date = date_parse(date_str)
     s3 = local_session(session_factory).client('s3')
     result = s3.get_object(Bucket=bucket, Key=key['Key'])
-    blob = StringIO(result['Body'].read())
+    blob = io.StringIO(result['Body'].read())
```

Comment:
did you test this? Is there a test case for this?
Since this change, I am getting `initial_value must be unicode or None, not str` errors here.